### DISCUSSION TITLE
Is Oozie still a component of TDP?

### DISCUSSION THREAD

**Discussion by jusstol** [TOSIT-IO/TDP#67] [Q&A] [answered]
Hi all,

When searching for "oozie" in all TOSIT-IO repositories, we can find some issues and merged PRs aiming to remove Oozie :
- https://github.com/TOSIT-IO/tdp-collection/issues/277
- https://github.com/TOSIT-IO/tdp-getting-started/issues/14
- https://github.com/TOSIT-IO/tdp-collection/pull/421
- https://github.com/TOSIT-IO/tdp-getting-started/pull/15

And considering the recent work on Hue :
- https://github.com/TOSIT-IO/tdp-collection-extras/pull/89
- https://github.com/TOSIT-IO/tdp-collection-prerequisites/pull/60
- https://github.com/TOSIT-IO/tdp-getting-started/pull/240

I come to wonder if Oozie is back in the game or if Hue isn't installing it either. If Oozie is no longer offered, is there an equivalent within TDP? Thanks a lot for your work.

**Comment by gonzaloetjo** [comment, marked as answer]
We are currently evaluating whether Airflow could serve as a suitable replacement. In the coming weeks, we will update the Airflow role ([related Issue in tdp-collections-extra](https://github.com/TOSIT-IO/tdp-collection-extras/issues/105)) and test connectors for Hive and Spark, as Oozie is not currently working with TDP.
